File: fizzBuzz.py
# -*- coding: utf-8 -*-
"""
Created on Fri Mar 27 19:23:24 2020

@author: paulb
"""
#Python FizzBuzz
for x in range (1, 101):
    fizz = (x % 3 == 0)
    buzz = (x % 5 == 0)
    if fizz & buzz:
        print ('FizzBuzz')
    elif fizz:
        print ('Fizz')
    elif buzz: 
        print ('Buzz')
    else:
        print (x)
        

File: fizzBuzz.r
fizzBuzz <- function(f){
  if (f%%5 == 0 && f%%3 == 0) {
    print('FizzBuzz')
  }
  else if(f%%5 == 0) {
    print('Buzz')
  }
  else if(f%%3 == 0) {
    print('Fizz')
  }
  else {
    print(f)
  }
}
sapply(seq(from = 1, to = 100, by = 1), fizzBuzz)